### PR TITLE
Migrate a few more files to Kotlin

### PR DESCRIPTION
**Pull request by gino-m**
Towards #1487

@JSunde PTAL?

### CODE REVIEW

File: ground/src/main/java/com/google/android/ground/persistence/local/room/fields/EntityState.kt [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    @TypeConverter
+    fun toInt(value: EntityState?): Int {
+      return toInt(value, UNKNOWN)
+    }
+
+    @JvmStatic
+    @TypeConverter
+    fun fromInt(intValue: Int): EntityState {
+      return fromInt(values(), intValue, UNKNOWN)
+    }
```
Comment (JSunde) [inline ground/src/main/java/com/google/android/ground/persistence/local/room/fields/EntityState.kt:42]: nit: these could be single expression functions

File: ground/src/main/java/com/google/android/ground/persistence/local/room/fields/MultipleChoiceEntityType.kt [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    fun toInt(value: MultipleChoiceEntityType?): Int {
+      return toInt(value, UNKNOWN)
+    }
+
+    @JvmStatic
+    @TypeConverter
+    fun fromInt(intValue: Int): MultipleChoiceEntityType {
+      return fromInt(values(), intValue, UNKNOWN)
+    }
+  }
```
Comment (JSunde) [inline ground/src/main/java/com/google/android/ground/persistence/local/room/fields/MultipleChoiceEntityType.kt:62]: ditto

File: ground/src/main/java/com/google/android/ground/persistence/local/room/fields/MutationEntitySyncStatus.kt [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    @TypeConverter
+    fun toInt(value: MutationEntitySyncStatus?): Int {
+      return toInt(value, UNKNOWN)
+    }
+
+    @JvmStatic
+    @TypeConverter
+    fun fromInt(intValue: Int): MutationEntitySyncStatus {
+      return fromInt(values(), intValue, UNKNOWN)
+    }
```
Comment (JSunde) [inline ground/src/main/java/com/google/android/ground/persistence/local/room/fields/MutationEntitySyncStatus.kt:63]: ditto

File: ground/src/main/java/com/google/android/ground/persistence/local/room/fields/MutationEntityType.kt [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    @TypeConverter
+    fun toInt(value: MutationEntityType?): Int {
+      return toInt(value, UNKNOWN)
+    }
+
+    @JvmStatic
+    @TypeConverter
+    fun fromInt(intValue: Int): MutationEntityType {
+      return fromInt(values(), intValue, UNKNOWN)
+    }
```
Comment (JSunde) [inline ground/src/main/java/com/google/android/ground/persistence/local/room/fields/MutationEntityType.kt:71]: ditto

**Review by JSunde — APPROVED** [review]

**Comment by gino-m** [comment]
/gcbrun

@JSunde RFAL

**Comment by JSunde** [comment]
Looks great!

**Review by JSunde — APPROVED** [review]

**Pull request merged**